### PR TITLE
style apple icon

### PR DESCRIPTION
**Pull request by muratcan23**

### CODE REVIEW

File: src/icons/GoogleIcon.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 const GoogleIcon = (props:any) => {
   return (
     <svg
-      width="18"
-      height="18"
+      width="15"
```
Comment (kod-man) [inline src/icons/GoogleIcon.tsx:4]: why is this one 15 and the others 16? Is there any specific reason?

**Review by kod-man — CHANGES_REQUESTED** [review]

File: src/icons/FacebookIcon.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 const FacebookIcon = (props:any) => {
   return (
     <svg
-      width="18"
-      height="18"
+      width="16"
+      height="16"
       viewBox="0 0 18 18"
```
Comment (kod-man) [inline src/icons/FacebookIcon.tsx:6]: I think you have to change viewBox 0 0 16 16

File: src/icons/GoogleIcon.tsx [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+      height="15"
       viewBox="0 0 18 18"
```
Comment (kod-man) [inline src/icons/GoogleIcon.tsx:6]: same

File: src/icons/GoogleIcon.tsx (hunk 1 above)
Comment (muratcan23) [reply src/icons/GoogleIcon.tsx:4]: yes, I tried 15-16-18 for each one, from my point of view, it seems more likely to resemble the original ones in these sizes.

**Review by kod-man — APPROVED** [review]

**Pull request merged**